status-im/status-react
# [#18512] incomplete list of collectibles

fixes #18512

### Summary

This PR shows collectibles having images in its `:collection-data`.
So we are still not listing some collectibles, but the reason is status-go returns them without any image (sometimes just a name).

This PR also adds partial support to display SVG collectibles, but they are still not animated, so this PR is not closing:
-  #18527

### Review notes

Future steps will be align with status-go team to investigate the reason of returning some collectibles empty.


#### Platforms

- Android
- iOS

### Steps to test


- Open Status
- Navigate to wallet tab
- Add an account owning collectibles
- Check the collectibles listed, we should display more than before, also we should be able to press on them and check their details.

status: ready
